src/integTest/kotlin/io/snyk/plugin/snykcode/core/IgnoreInfoHolderPlatformTestCase.kt

```diff
@@ -14,6 +15,11 @@ import java.io.FileNotFoundException
 // See: https://plugins.jetbrains.com/docs/intellij/light-and-heavy-tests.html
 class IgnoreInfoHolderPlatformTestCase : HeavyPlatformTestCase() {
 
+    override fun setUp() {
+        super.setUp()
+        ModuleRootModificationUtil.addContentRoot(module, project.basePath!!)
```

Comment:
@ArtsiomCh, why we need this change?

Reply:
@pavel-snyk HeavyPlatformTestCase does not create ContentRoot for default project. And now we use IntelliJ api to iterate over project files that internally relay on ContentRoot setting.

Reply:
got it, thanks for clarification.